### DISCUSSION TITLE
VABS input files

### DISCUSSION THREAD

**Discussion by BeatriceRimoldi** [wenbinyugroup/.github#7] [VABS Related Discussions] [answered]
Hi,
Following the instructions written here [Examples — PreVABS 1.2 documentation (purdue.edu)](https://urldefense.com/v3/__https:/pages.github.itap.purdue.edu/wenbinyugroup/prevabs_doc/examples.html__;!!Pf7i9cU!l7QFabz4-e62oR5TCljWo7Wx3ajQuhoXEcoAjayynwDrNUBWJDu26ysGlrvaKXP2VRG4yFc8ny-WNhedLEH7MxtzFg$) I am using preVABS to create the geometry of the section I want to analyze, I prepare the files basepoints.dat, baselines.xml, materials.xml, layups.xml and a main file .xml that I run on prevabs from the command prompt and it opens gmsh with the section I created mesh included. 
From here I would like to use VABS to run the simulation and get a .k file with the cross-section properties. Is this correct?
In the VABS folder that I downloaded from the Altair marketplace there is a 3cells.dat input file as an example in which, I think I understand, is contained all the information that in the examples instead is in the main.xml file.
If I try to drag and drop this file 3cells.dat to VABS.exe I get the .k file with the results I would be looking for.
Among the files that prevabs creates for me, when I run my sections or the example sections, I can't find any .dat that is suitable to be used with VABS I don't understand what I'm doing wrong in the process, would anyone be able to help me?
I hope I have been clear, but I welcome any suggestions you may have on the subject.
thanks

**Comment by wenbinyu-PU** [comment, marked as answer]
There is not a fixed extension for VABS input file. PreVABS might create a file called *.sg. You can use that file as the input.
